src/components/CameraPermissions/Recover/style.scss

```diff
@@ -22,12 +19,9 @@
   margin: 0 0 8 * $unit-small;
 }
 
+/* Camera Permission "Recovery" header pill, has subtle differences to the Theme header pill */
```

Comment:
I don't remember what the design looks like. I wonder if makes sense to make these two pills consistent?

Reply:
Have addressed this and have made adjustments to make the pill on both screens use the same style.